.travis.yml

```diff
@@ -0,0 +1,40 @@
+env:
+  global:
+    - VERSION=1.${TRAVIS_BUILD_NUMBER}
+  matrix:
+    - ARCH=amd64
+      OS=linux
+      DOCKER_BUILD=gini/trello-dashboard:${VERSION}
+      DOCKERFILE=Dockerfile.amd64
+    - ARCH=arm
+      OS=linux
+      DOCKER_BUILD=gini/trello-dashboard:rpi-${VERSION}
+      DOCKERFILE=Dockerfile.rpi
+
+sudo: required
+
+services:
+  - docker
+
+language: go
+
+before_install:
+  - go get github.com/Sirupsen/logrus
+  - go get github.com/VojtechVitek/go-trello
+
+script:
+  - GOOS=$OS GOARCH=$ARCH go build -v -o "trello-dashboard_$OS-$ARCH" -i .
+  - docker build -t ${DOCKER_BUILD} -f ${DOCKERFILE} .
+#  - docker run --rm --privileged multiarch/qemu-user-static:register
+#  - docker run --rm -v $PWD:/usr/src/myapp -w /usr/src/myapp -v go:/go ${GOIMG} bash -c "cd /usr/src/myapp && go get -d -v -t && go test --cover -v ./... --run UnitTest && go build -v -o trello-dashboard"
+
+after_success:
+#  - docker run --rm --privileged multiarch/qemu-user-static:register
+
+deploy:
+  provider: script
+  script: scripts/deploy.sh
```

Comment:
Any tangible benefits from using a script instead of adding the 2 lines directly?